src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -147,11 +147,7 @@ protected void ProcessAssemblyAction (AssemblyDefinition assembly)
 			case AssemblyAction.AddBypassNGen:
 				// FIXME: AddBypassNGen is just wrong, it should not be action as we need to
 				// turn it to Action.Save here to e.g. correctly update debug symbols
```

Comment:
Should we also delete the comment now?

Reply:
I think that still wrong. The whole AddBypassNGen thing is weird :-) and has no tests.